README.md

```diff
@@ -1,4 +1,29 @@
-# Gateway API Inference Extension 
+# Gateway API Inference Extension
+
+This project offers tools for [AI Inference], enabling developers to build [AI Gateways and Inference Frameworks].
+
+[AI Inference]:https://research.ibm.com/blog/AI-inference-explained
+[AI Gateways and Inference Frameworks]:#concepts-and-definitions
+
+## Concepts and Definitions
+
+AI/ML is evolving rapidly, and [Inference] goes beyond basic networking to include complex traffic routing and optimizations. Below are key terms for developers:
+
+- **GIE**: Short for "Gateway API Inference Extension".
+- **Scheduler**: Manages routing in inference, basing decisions on routing rules (e.g., "model name") and `Metrics and Capabilities` from [Model Serving Platforms].
+- **Metrics and Capabilities**: Data provided by model serving platforms about performance, availability and capabilities to optimize routing. Includes things like [Prefix Cache] status or [LoRA Adapters] availability.
+- **Inference Framework**: The combination of a `Scheduler`, its associated `Metrics and Capabilities`, and an `AI Gateway` is considered an `Inference Framework`.
```

Comment:
I am a little confused by this concept for a couple of reasons:

1) I think "Inference Framework" is a very broad term that I would expect to include components other than the inference gateway (model servers for example), which is not what I assume the intention here.

2) My read is that this is defining a parallel concept to "Inference Gateway", which the next section already defines as a Proxy (or k8s gateway) + Inference Extension, with the "Inference Extension" being the component that encapsulates scheduling, load and status probing of model servers.

Reply:
Ok, let me pull it out for now and we'll think through it some more.

```suggestion
```